iphone/Maps/LocalizedStrings/sk.lproj/InfoPlist.strings

```diff
@@ -1,3 +1,8 @@
+/**
```

Comment:
Something is very wrong. Please do interactive rebase, remove all "strings regenerated" commits, and then generate strings again in a separate commit by `tools/unix/generate_localizations.sh`

Reply:
Yess i did that only. 
Ok will do it again may be i might have done something wrong.